src/services/bria/index.ts

```diff
@@ -300,6 +305,47 @@ export const OnChainService = (): IOnChainService => {
     }
   }
 
+  const rebalanceToHotWallet = async (
+    amount: BtcPaymentAmount,
+  ): Promise<UnsignedPsbt | OnChainServiceError> => {
+    const submitPayoutRequest = new SubmitPayoutRequest()
+    submitPayoutRequest.setWalletName(briaConfig.coldStorage.walletName)
+    submitPayoutRequest.setPayoutQueueName(
+      briaConfig.coldStorage.coldToHotRebalanceQueueName,
+    )
+    submitPayoutRequest.setDestinationWalletName(briaConfig.hotWalletName)
+    submitPayoutRequest.setSatoshis(Number(amount.amount))
+    submitPayoutRequest.setMetadata(
+      constructMetadata({ galoy: { rebalanceToHotWallet: true } }),
+    )
+
+    const triggerPayoutQueueRequest = new TriggerPayoutQueueRequest()
+    triggerPayoutQueueRequest.setName(briaConfig.coldStorage.coldToHotRebalanceQueueName)
+
+    try {
+      const submitPayoutResponse = await submitPayout(submitPayoutRequest, metadata)
+
+      await triggerPayoutQueue(triggerPayoutQueueRequest, metadata)
+
+      const payoutRequest = new GetPayoutRequest()
+      payoutRequest.setId(submitPayoutResponse.getId())
+      const getPayoutResponse = await getPayout(payoutRequest, metadata)
+
+      const foundPayout = getPayoutResponse.getPayout()
+      if (!foundPayout) return new PayoutNotFoundError()
+      const batchId = foundPayout.getBatchId()
```

Comment:
We may need to loop here - triggering the queue is not instant. The queue will get instantly scheduled but it still takes some time until it is processed.

An alternative would be to get the batch id from the `PayoutCommitted` event that should result from the queue getting processed.